src/components/structures/RoomSubList.js

```diff
@@ -255,10 +340,17 @@ const RoomSubList = createReactClass({
             chevron = (<div className={chevronClasses} />);
         }
 
-        const tabindex = this.props.isFiltered ? "0" : "-1";
         return (
-            <div className="mx_RoomSubList_labelContainer" title={ title } ref="header">
-                <AccessibleButton onClick={this.onClick} className="mx_RoomSubList_label" tabIndex={tabindex} aria-expanded={!isCollapsed}>
+            <div className="mx_RoomSubList_labelContainer" title={title} ref="header" onKeyDown={this.onHeaderKeyDown}>
+                <AccessibleButton
+                    onClick={this.onClick}
+                    className="mx_RoomSubList_label"
+                    tabIndex={0}
+                    aria-expanded={!isCollapsed}
+                    inputRef={this._headerButton}
+                    // cancel out role so this button behaves as the toggle-header of this group
+                    role="none"
```

Comment:
Just checking, so not `treeitem` as Marco suggested?

Reply:
Ah yes, that makes more sense :D

Reply:
This makes it a bit noisier but is more semantically correct so will commit it

Reply:
so with this not being role=none it gets counted as an "item" inside that section so if you have 1 room in a section it'll say x out of 2. But is semantically correct 🤷‍♂

Reply:
Hmm, would be interesting to know whether browsers and platforms consistently behave like this, and from Marco if this is the behaviour he expected. I agree it's good to go with this now and get feedback on it.